src/js/components/Browser.react.js

```diff
@@ -194,20 +194,51 @@ class Browser extends React.Component<Props, State> {
           contextSelector: context,
         });
       }
-    } else {
+    }
+  };
+
+  highlightWarningElements = () => {
```

Comment:
Perhaps we should also rename the existing function `highlightElements` so that it is clear that is highlights the selectors coming from the editor.